lib/html_parser.dart

```diff
@@ -800,3 +804,45 @@ class HtmlParser {
     return false;
   }
 }
+
+class DetailsWidget extends StatefulWidget {
```

Comment:
Thank you for this widget. Rather than including the `DetailsWidget` class in the `lib/html_parser.dart` file, could you move it to `lib/widgets/details_widget.dart`? This will help keep html_parser.dart from getting too bloated when I start to add additional custom widgets.

Reply:
Yeah, i'll do it asap.